docs/data-sources/groups.md

```diff
@@ -37,6 +37,7 @@ The following arguments are supported:
 * `display_names` - (Optional) The display names of the groups.
 * `object_ids` - (Optional) The object IDs of the groups.
 * `return_all` - (Optional) A flag to denote if all groups should be fetched and returned.
+* `security_enabled` - (Optional) A flag to denote if only `security_enabled=true` groups should be returned.
```

Comment:
```suggestion
* `security_enabled` - (Optional) Whether the groups are security-enabled.
```

Reply:
I like the previous wording - It matches the boolean flag wording of `return_all` - But I understand if the english is a bit crazy.

Reply:
That's fine, I was thinking to align the wording more with the `azuread_group` DS but this is sufficiently different 👍